utils/esr_peer_info.cpp

```diff
@@ -358,13 +358,16 @@ namespace
 
 
     std::string
-    pad_str_to_mod_64(const std::string &parameter)
+    pad_str_to_mod_64(std::string parameter)
     {
-        const size_t REQUIRED_MOD{64};
-        const size_t PADDING_REQUIRED = (REQUIRED_MOD - parameter.size() % REQUIRED_MOD);
-        std::string result{parameter};
-        result.insert(result.size(), PADDING_REQUIRED, '0');
-        return result;
+        static const size_t REQUIRED_SIZE_MULTIPLE{64};
```

Comment:
please stop using statics... move to unnamed namespaces.
You incur a performance hit for every call of a function.

Reply:
ok